mw-webapp/src/component/dropdown/dropdownMenuItem/DropdownMenuItem.tsx

```diff
@@ -0,0 +1,57 @@
+import styles from "src/component/dropdown/DropdownMenuItem/DropdownMenuItem.module.scss";
+
+/**
+ * Option type for a select component
+ */
+export interface DropdownMenuItemType {
+
+  /**
+   * Item`s id. Should be unique
+   */
+  id: string;
+
+  /**
+   * Item`s value
+   */
+  value: string;
+
+  /**
+   * Item`s onClick
+   */
+  onClick: () => void;
+
+  /**
+   * Item`s visible state
+   */
+  isVisible: boolean;
```

Comment:
could it be optional prop with visible by default?

Reply:
I added isVisible?: boolean; to make the isVisible property optional. If this property is absent, isVisible = true by default